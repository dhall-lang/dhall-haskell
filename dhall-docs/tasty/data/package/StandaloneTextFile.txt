This is a text file that is not referenced by any dhall file.

+---------------------------------------------------+
| It should be displayed as <pre>preformatted</pre> |
|   text, with horizontal and vertical whitespace   |
|          preserved, to keep it readable.          |
+---------------------------------------------------+
