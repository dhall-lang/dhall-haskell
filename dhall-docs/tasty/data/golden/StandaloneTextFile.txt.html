<!DOCTYPE HTML>
<html>
<head>
<title>/StandaloneTextFile.txt</title>
<link rel="stylesheet" type="text/css" href="index.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap">
<script src="index.js">
</script>
<meta charset="UTF-8">
</head>
<body>
<div class="nav-bar">
<img class="dhall-icon" alt="Dhall logo." src="dhall-icon.svg">
<p class="package-title">test-package</p>
<div class="nav-bar-content-divider">
</div>
<a id="switch-light-dark-mode" class="nav-option">Switch Light/Dark Mode</a>
</div>
<div class="main-container">
<h2 class="doc-title">
<span class="crumb-divider">/</span>
<a href="index.html">test-package</a>
<span class="crumb-divider">/</span>
<span class="title-crumb" href="index.html">StandaloneTextFile.txt</span>
</h2>
<a class="copy-to-clipboard" data-path="/StandaloneTextFile.txt">
<i>
<small>Copy path to clipboard</small>
</i>
</a>
<br>
<h3>Source</h3>
<div class="source-code">
<pre>This is a text file that is not referenced by any dhall file.

+---------------------------------------------------+
| It should be displayed as &lt;pre&gt;preformatted&lt;/pre&gt; |
|   text, with horizontal and vertical whitespace   |
|          preserved, to keep it readable.          |
+---------------------------------------------------+
</pre>
</div>
</div>
</body>
</html>